processing/src/main/java/org/apache/druid/guice/PropertiesModule.java

```diff
@@ -69,8 +69,8 @@ public void configure(Binder binder)
 
         if (stream != null) {
           log.info("Loading properties from %s", propertiesFile);
-          try {
-            fileProps.load(new InputStreamReader(stream, StandardCharsets.UTF_8));
+          try (InputStreamReader isr = new InputStreamReader(stream, StandardCharsets.UTF_8)) {
+            fileProps.load(isr);
```

Comment:
This file has been changed on `master`. On master this cannot be rewritten since the stream is being consumed after it has been closed, which will throw an exception. Also, these constructions don't help: https://github.com/apache/incubator-druid/blob/master/processing/src/main/java/org/apache/druid/guice/PropertiesModule.java#L81 
Curious in which case `.exists()` lies 🤔